pkgs/tools/wayland/proycon-wayout/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv
+, lib
+, fetchFromSourcehut
+, meson
+, wayland-protocols
+, wayland
+, cairo
+, pango
+, scdoc
+, ninja
+, cmake
+, pkg-config
+, wayland-scanner
+}:
+
+stdenv.mkDerivation rec {
+  pname = "proycon-wayout";
+  version = "0.1.3";
+
+  src = fetchFromSourcehut {
+    owner = "~proycon";
+    repo = "wayout";
+    rev = version;
+    sha256 = "sha256-pxHz8y63xX9I425OG0jPvQVx4mAbTYHxVMMkfjZpURo=";
+  };
+
+  depsBuildBuild = [ pkg-config ];
+  nativeBuildInputs = [ scdoc ninja meson cmake pkg-config wayland-scanner ];
+  buildInputs = [ wayland-protocols wayland cairo pango wayland-scanner ];
+
+  fixupPhase = ''
```

Comment:
Overwriting this phase is forbidden. Use postFixup. Aldo why do we must rename the package? Do people actually use both?

Reply:
Needing both is very realistic, as they both are common utilities targeting wayland/wlroots users. They perform completely unrelated functions, just happen to share a name.

(I will rename to postFixup)

Reply:
fixupPhase replaced with postFixup.

Reply:
There should be something about that you shouldn't overwrite phases.